### PR TITLE
Put frontend assets to the lpvs container

### PR DESCRIPTION
**Pull request by tiokim**
# Description

This PR is intended to provide frontend and backend services in the lpvs container.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?
1. Update environment variables of `docker-compose.yml` file.
2. Build the lpvs container : `docker compose build`
3. Run containers: `docker compose up -d`
4. See the lpvs home page with any browser: `http://localhost:7896`

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by tiokim** [comment]
@kyudori Can you check if this PR works well?

**Comment by kyudori** [comment]
> @kyudori Can you check if this PR works well?

@tiokim  Yes.
I checked frontend page working successfully (docker container) :
1. Google social login.
2. Modifying user information on the user/info page.
3. Logout.

All three processes are working successfully

Docker

**Comment by kyudori** [comment]
> @kyudori Can you check if this PR works well?
However, when refreshing the user/info page with an F5, JSON-format data is displayed instead of the front-end page.
image

**Comment by tiokim** [comment]
> > @kyudori Can you check if this PR works well?
> 
> @tiokim However, when refreshing the user/info page with an F5, JSON-format data is displayed instead of the front-end page. image

@kyudori Please verify it again.

**Comment by kyudori** [comment]
> @kyudori Please verify it again.

@tiokim Front page is displayed successfully even after refreshing F5.

File: docker-compose.yml [hunk 1]
```diff
@@ -10,20 +10,26 @@ services:
     ports:
       - "7896:7896"
     environment:
+#      - frontend.main-page.url=http://localhost:3000
+#      - cors.allowed-origin=http://localhost:3000
```
Comment (m-rudyk) [inline docker-compose.yml:14]: Please double check, maybe this options can be uncommented as default fonfiguration?

Comment (tiokim) [reply docker-compose.yml:14]: I left them for reference. They are not necessary if lpvs backend and frontend run in a single container.

**Review by MoonkiHong — APPROVED** [review]
+1

**Pull request merged**